app/views/publishers/home.html.slim

```diff
@@ -18,34 +18,15 @@
         - else
           div= I18n.t("publishers.status_to_receive_payments")
           ul
-            - if current_publisher.bitcoin_address.blank?
-              li= I18n.t("publishers.status_bitcoin_address_required")
-            - if !current_publisher.legal_form_completed?
-              li= I18n.t("publishers.status_legal_form_required")
+            - if current_publisher.uphold_status != :verified #ToDo: Further breakdown
+              li= I18n.t("publishers.status_connect_to_uphold_required")
```

Comment:
Could we make this link, for example to the publisher uphold authorize path? It would help clarify where to proceed.